src/domain/Popovers/DropdownMenu/subcomponents/Section.tsx

```diff
@@ -92,10 +95,55 @@ class Section extends React.PureComponent<ISectionProps, never> {
   private get content () {
     const {
       component,
-      componentProps,
       leftComponent,
       rightComponent,
       text,
+      iconType,
+      icon,
+      secondaryText
+    } = this.props
+
+    if (component) {
+      return (
+        <StyledSection {...this.styleProps}>
+          {component}
+        </StyledSection>
+      )
+    }
+    const hasIcon = icon && iconType
```

Comment:
Would it be more appropriate for the `icon` prop to be like:
```
iconProps?: {
  icon: FontAwesomeIconValue
  iconType: 'solid' | 'regular' | 'light' | 'duotone'
}
```
Which will force people to pass in both values when they use the component. I don't think its super intuitive that if you don't use the optional `iconType` prop, your icon won't show up.